src/pages/Models/Models.js

```diff
@@ -21,7 +22,19 @@ function pluralize(value, string) {
 
 export default function Models() {
   const { blocked, alert, running } = useSelector(getGroupedModelStatusCounts);
-  const [groupedBy, setGroupedBy] = useState("status");
+
+  const location = useLocation();
+  // Grab filter from 'groupby' query in URL and assign to variable
+  // If it doesn't exist, fall back to grouping by status
+  const groupFilterFromQueryString =
+    location.search.split("groupby=")[1] || "status";
+  // Set variable as default state value
+  const [groupedBy, setGroupedBy] = useState(groupFilterFromQueryString);
+  // Listen for back button click and update groupedBy state accordingly
+  window.onpopstate = function() {
```

Comment:
You shouldn't redefine a global in a component as everywhere will use this method now for `onpopstate`.